src/rpc/blockchain.cpp

```diff
@@ -964,7 +971,17 @@ static RPCHelpMan getblock()
         return strHex;
     }
 
-    return blockToJSON(block, tip, pblockindex, verbosity >= 2);
+    TxVerbosity block_to_json_verbosity;
+
+    if (verbosity <= 1) {
+      block_to_json_verbosity = TxVerbosity::SHOW_TXID;
```

Comment:
indentation is wrong

Reply:
(Looks correct to me now; please mark as resolved 🙏 )